Szymon/src/test/java/zadanie6/page/ForgotPasswordPage.java

```diff
@@ -0,0 +1,55 @@
+package zadanie6.page;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class ForgotPasswordPage extends LeftNavigationMenu{
```

Comment:
Ciekawy pomysl z tym rozszerzaniem.
Jakbys wprowadzil wiecej porzadku z poziomu tetow, to byloby troche bardziej czytelne.

Metody testowe musza byc czytelne tak, zeby ktos kto czyta, wiedzial jaki scenariusz jest testowany.